snapshot/git/gitfiler/setup.go

```diff
@@ -43,26 +39,3 @@ func NewSingleRepoCheckouter(repoIniter PooledRepoIniter,
 	pool := NewSingleRepoPool(repoIniter, stat, doneCh)
 	return NewCheckouter(pool, stat)
 }
-
-// A Checkouter that creates a new repo with git clone --reference for each checkout
-func NewRefRepoCloningCheckouter(refRepoIniter PooledRepoIniter,
```

Comment:
This is still called in closed source (for mac-based workers I believe, which meh, but worth calling out that this should trigger deprecation of that completely)

Reply:
Ah - could also just leave it in & change the implementation of the refCloner so that it takes a path rather than a *temp.TempDir

Reply:
I think I am gonna get rid of macbook filer compatibility altogether, fyi

Reply:
ah missed this when I merged. fine with me, can remove all this stuff with that removal.